lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -179,6 +196,9 @@ export default function AssignmentActivity() {
                   </span>
                 )
               );
+            case 'auto_grading_grade':
+              // TODO Return a formatted component
+              return `${stats.auto_grading_grade}%`;
```

Comment:
The plan is to use a component from https://github.com/hypothesis/lms/pull/6658 here.